src/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ if (“${PROJECT_SOURCE_DIR}” STREQUAL “${CMAKE_SOURCE_DIR}”)
     # Set some default BLT options before loading BLT only if not included in
     # another project
     if (NOT BLT_CXX_STD)
-        set(BLT_CXX_STD "c++11" CACHE STRING "")
+        set(BLT_CXX_STD "c++14" CACHE STRING "")
```

Comment:
Please update our `RELEASE-NOTES` that the default is now `c++14` and that `c++14` is required for or RAJA and Umpire TPLs.